docs/pages/setup/admin/adding-nodes.mdx

```diff
@@ -74,36 +112,53 @@ auth_service:
 ```
 </Details>
 
-### Using Node invitation tokens
+### Start your Node with the invite token and CA pin
+
+The CA pin at the needs to be passed to new Nodes when they're
```

Comment:
I'm having a hard time parsing "The CA pin at the needs to be passed ...". Is there a typo here?

Reply:
There is—fixed